ext/tiny_tds/client.c

```diff
@@ -392,6 +421,7 @@ static VALUE rb_tinytds_connect(VALUE self, VALUE opts) {
     } else {
       cwrap->identity_insert_sql = "SELECT CAST(SCOPE_IDENTITY() AS bigint) AS Ident";
     }
+    dbsetinterrupt(cwrap->client, check_interrupt, handle_interrupt);
```

Comment:
Is it possible for a connection to time out between `dbopen` above and this line? I wonder if this is like the first thing we should set up, before trying to send any db commands like `dbuse`. Maybe right after `dbsetuserdata`?